### ISSUE TITLE
`<not implemented for provider docker>`

### ISSUE THREAD

**Issue by ghost** [kuboj/vagrant-portinfo#2]
Unable to use with docker as a provider :(

**Comment by kuboj** [comment]
Hello. I'll look into it, currently I don't know how much effort it would require.

**Comment by ghost** [comment]
I did a workaround with parsing the vagrant up output to grab port numbers. Has the caveat that subsequent vagrant up's do not contain port numbers. A solution that queries docker container data would be preferable of course.

**Comment by kuboj** [comment]
Yes, you're right. Seems like that parsing output of `docker port` could be the way. The question is, if it provides information about _complete_ port configuration, both for mapping set in Dockerfile and command line set.
